src/Native/Runtime/gcrhenv.cpp

```diff
@@ -948,9 +949,20 @@ void RedhawkGCInterface::DestroyTypedHandle(void * handle)
 
 void* RedhawkGCInterface::CreateTypedHandle(void* pObject, int type)
 {
-    return (void*)::CreateTypedHandle(g_HandleTableMap.pBuckets[0]->pTable[GetCurrentThreadHomeHeapNumber()], (Object*)pObject, type);
+    return (void*)GCHandleTableUtilities::GetGCHandleTable()->CreateHandleOfType(g_HandleTableMap.pBuckets[0]->pTable[GetCurrentThreadHomeHeapNumber()], (Object*)pObject, type);
 }
 
+#ifndef FEATURE_REDHAWK
```

Comment:
`FEATURE_REDHAWK` means CoreRT. Since this file is CoreRT-specific, it does not make sense to have code under `ifndef FEATURE_REDHAWK` in it. Could you please delete?